content/logs/_index.md

```diff
@@ -235,10 +237,12 @@ logs:
 
 ### Multi-line
 
-If your logs are not sent in JSON and you want to aggregate several lines into one single log, you can configure the agent to detect a new log using a specific regex pattern instead of having one log per line. This is accomplished by using the `log_processing_rules` parameter in your configuration file with the **multi_line** `type`.
+If your logs are not sent in JSON and you want to aggregate several lines into one single log, configure the Datadog Agent to detect a new log using a specific regex pattern instead of having one log per line.  
```

Comment:
"single log" -> "single entry"

Reply:
Additionally, another case where we call it Datadog Agent here but Log Agent elsewhere. We should pick one and be consistent.

Reply:
:+1: